config/taxonomy/catalog.structs.schema.json

```diff
@@ -51,20 +55,42 @@
                                 "type": "string",
                                 "description": "name of column or component"
                             },
-                            "tags": { "$ref": "catalog.values.schema.json#/definitions/valid_metadata"}
+                            "tags": { "$ref": "#/definitions/tags" }
                         },
-                        "required":["name"]
+                        "required": ["name"]
                     }
                 }
             },
-            "required":["name"],
-            "additionalProperties": false
-        }
-    },
+            "required": ["name"],
+            "additionalProperties": {
+                "creationTime": {
+                    "type": "string",
+                    "description": "Time when the asset was created"
+                },
+                "CreatedBy": {
+                    "type": "string",
+                    "description": "Owner Id for the user who created the asset"
+                },
+                "updationTime": {
```

Comment:
updation is not a word I don't think.  Did you mean updateTime?

Reply:
Yes. Let me correct it.

Reply:
Done